community-operators/opentelemetry-operator/0.27.0/manifests/opentelemetry-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    control-plane: controller-manager
+  name: opentelemetry-operator-controller-manager-metrics-service
+spec:
+  ports:
+  - name: https
+    port: 8443
+    targetPort: https
+  selector:
+    control-plane: controller-manager
+status:
```

Comment:
This looks odd, but I assume this is just how the operator-sdk is generating this file, right?

Reply:
Yes, definitely strange to see status in manifest file but you are right it is what operator-sdk generated. Let me take another look to make sure.

Reply:
Yes, this was generated by the sdk bundle command.